tests/sources/test_github.py

```diff
@@ -1230,6 +1273,42 @@ async def test_fetch_repos_with_unauthorized_exception():
                 pass
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "repo_type, repos, expected_repos",
+    [
+        ("organization", "*", [MOCK_REPO_1_DOC, MOCK_REPO_2_DOC]),
+        ("other", "*", [MOCK_REPO_3_DOC, MOCK_REPO_4_DOC]),
+        ("organization", "org_1/repo_1", [MOCK_REPO_1_DOC]),
+        ("other", "user_1/repo_3", [MOCK_REPO_3_DOC]),
+    ],
+)
```

Comment:
Nit: it might be worth adding an example with multiple specified `repos`.
(It supports comma-separated lists, right?)

Reply:
Added here: https://github.com/elastic/connectors/pull/2412/commits/e3e841dc349adfafcb0b0d954a3ecc4eb72f5e5b